tests/test_config_probe.py

```diff
@@ -1,24 +1,40 @@
 import unittest
 
 import os
-from config_probe import probe, fake_probe
+/from config_probe import probe, fake_probe
```

Comment:
I shouldn't commit before coffe :\ fixed

On Thu, Jun 30, 2016 at 9:07 AM Martin Roy <notifications@github.com> wrote:

> In tests/test_config_probe.py
> <https://github.com/internap/python-config-probe/pull/12#discussion_r69127896>
> :
>
> > @@ -1,24 +1,40 @@
> >  import unittest
> >
> >  import os
> > -from config_probe import probe, fake_probe
> > +/from config_probe import probe, fake_probe
>
> that's probably a typo :P
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/internap/python-config-probe/pull/12/files/4484b5b7397d6873067d6499cdbd9f2eab96dba6#r69127896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABhPoJ0w9ZCygTipjeL614rBhHMltjiEks5qQ7-SgaJpZM4I6Izf>
> .
>